### ISSUE TITLE
script to restart device

### ISSUE THREAD

**Issue by afatsoum** [amelchio/eternalegypt#26]
Since restarting the device is a standard built-in function, would be great to have this included as well.

Is it possible to be able to call the restart function from python script?

I am trying to obtain a new IP from the LTE network but it seems reconnecting is not enough, a full restart is required.

I am using LB2120 with latest update as of this note.

Firmware Version: M18QW_v07.05.170721
App Version: NTG9X07W_13.05.05.00
Web App Version: LBHDATA_03.03.103.175
Bootloader Version: Bootloader Ver: v1.0.0
Hardware Version: 08

**Comment by amelchio** [comment]
That should be possible to add. Are you running Home Assistant or using the library stand-alone?

**Comment by afatsoum** [comment]
not using home assistant, stand alone library direct with lb2120 modem.

**Comment by afatsoum** [comment]
if it helps, it should work with the following snippet

**add to eternalegypt.py**
```
    @autologin
    async def router_reset(self):
        """Do an device restart."""
        async with self._config_call('general.routerReset', 1) as response:
            _LOGGER.debug("router reset %d", response.status)
```

**example call**

```
#!/usr/bin/env python3

"""Example file for eternalegypt library."""

import sys
import asyncio
import aiohttp
import logging
logging.basicConfig(level=logging.DEBUG)

import eternalegypt

async def restart():
    """Example of printing the current upstream."""
    jar = aiohttp.CookieJar(unsafe=True)
    websession = aiohttp.ClientSession(cookie_jar=jar)

    try:
        modem = eternalegypt.Modem(hostname=sys.argv[1], websession=websession)
        await modem.login(password=sys.argv[2])

        await modem.router_reset()

        await modem.logout()
    except eternalegypt.Error:
        print("Could not login")

    await websession.close()

if len(sys.argv) != 3:
    print("{}: <netgear ip> <netgear password>".format(sys.argv[0]))
else:
    asyncio.get_event_loop().run_until_complete(restart())
```

**Comment by amelchio** [comment]
Thank you, I now released that as v0.0.13 (renamed to `router_restart()`).
